ts_src/address.ts

```diff
@@ -294,3 +294,31 @@ function toConfidentialSegwit(
   const data = toOutputScript(address, network);
   return toBlech32(data, blindingKey, network.blech32);
 }
+
+/**
+ * A quick check used to verify if a string could be a confidential segwit address.
+ * @param address address to test.
+ */
+function isConfidentialSegwit(address: string): boolean {
+  if (address.length !== 80) return false;
+  if (address.startsWith('Az')) return true;
+  return false;
+}
```

Comment:
This util function MUST be fixed: for the moment it helps only to know if an address is a wrapped segwit for regtest network!
You can take inspiration from [go-elements](https://github.com/vulpemventures/go-elements/blob/521a5fc29d945e073e6f676d01c172d11ecf978b/address/address.go#L515) (isBech32, isBlech32, etc) for this